examples/guestbook/all-in-one/frontend.yaml

```diff
@@ -16,24 +16,25 @@ spec:
     app: guestbook
     tier: frontend
 ---
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
   name: frontend
   # these labels can be applied automatically
   # from the labels in the pod template if not set
-  labels:
-    app: guestbook
-    tier: frontend
+  # labels:
+  #   app: guestbook
+  #   tier: frontend
 spec:
   # this replicas value is default
   # modify it according to your case
   replicas: 3
```

Comment:
If you view the file it looks fine https://github.com/janetkuo/kubernetes/blob/automated-cherry-pick-of-%2323408-upstream-release-1.2/examples/guestbook/all-in-one/frontend.yaml

I think it's github rendering issue?